### ISSUE TITLE
Point Release/Tagged Release with recent changes

### ISSUE THREAD

**Issue by abuyoyo** [WyriHaximus/HtmlCompress#92]
Is it possible to make a tagged release with recent changes - that will be available on Packagist?
I'd really like to deploy the recent changes into one of my projects. Without having to revert to `vcs:{}`, `dev-develop`, a fork, etc.

**Comment by WyriHaximus** [comment]
Got a long weekend, I'll try to get all open PR's in before over the next few days and tag a release then :)

**Comment by WyriHaximus** [comment]
Just tagged a new release: https://github.com/WyriHaximus/HtmlCompress/releases/tag/4.0.0

**Comment by abuyoyo** [comment]
Thanks :)
Is it possible to reduce the composer.json PHP version requirement to `>=7.2`?
My local machine is still on 7.2
And my production installs are 7.2~7.3 at the moment

**Comment by WyriHaximus** [comment]
TL;DR: No. But. 

For me v4 was going to be a version to experiment with PHP 7.4 features turning it into a sort of LTS version for PHP for those who can't upgrade to PHP 8 hitting later this year. However I'm sure I can find a way to get this into v3.

**Comment by WyriHaximus** [comment]
Work in Progress PR is up at: https://github.com/WyriHaximus/HtmlCompress/pull/96
